sql/core/benchmarks/CollationBenchmark-jdk21-results.txt

```diff
@@ -1,54 +1,63 @@
+UNICODE                                              18863          18867           6          0.0      188625.3       0.2X
+UTF8_BINARY                                           4894           4901          11          0.0       48936.8       1.0X
+UNICODE_CI                                           19595          19598           4          0.0      195953.0       0.2X
 
-OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
+OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1021-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - hashFunction:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY_LCASE                                  4824           4824           0          0.0       48243.7       1.0X
-UNICODE                                           69416          69475          84          0.0      694158.3       0.1X
-UTF8_BINARY                                        3806           3808           2          0.0       38062.8       1.3X
-UNICODE_CI                                        60943          60975          45          0.0      609426.2       0.1X
+UTF8_BINARY_LCASE                                  5011           5013           2          0.0       50113.1       1.0X
+UNICODE                                           68309          68319          13          0.0      683094.7       0.1X
+UTF8_BINARY                                        3887           3887           1          0.0       38869.8       1.3X
+UNICODE_CI                                        56675          56686          15          0.0      566750.3       0.1X
 
-OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
+OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1021-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - contains:     Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY_LCASE                                 11979          11980           1          0.0      119790.4       1.0X
-UNICODE                                            6469           6474           7          0.0       64694.8       1.9X
-UTF8_BINARY                                        7253           7253           1          0.0       72528.3       1.7X
-UNICODE_CI                                       319124         319881        1070          0.0     3191244.0       0.0X
+UTF8_BINARY_LCASE                                 10534          10534           1          0.0      105336.8       1.0X
+UNICODE                                            5835           5836           2          0.0       58348.9       1.8X
```

Comment:
Performance of `contains` has gotten a little better since last this was run. Nice!